### ISSUE TITLE
[Accessibility] PeoplePicker: Doesn't Announce 'No results found'

### ISSUE THREAD

**Issue by HalcyonPride** [OfficeDev/office-ui-fabric-react#7991]
### Environment Information 
 
- __Package version(s)__: Latest
- __Browser and OS versions__: Edge; Windows 10 Enterprise, version 1809

### Please provide a reproduction of the bug in a codepen:
https://developer.microsoft.com/en-us/fabric#/components/peoplepicker
 

 
#### Actual behavior:
 
When the list of Suggested People is empty, Narrator doesn't announce anything.
 
#### Expected behavior:
 
Narrator should announce 'No results found'.

#### Priorities and help requested:
 
Are you willing to submit a PR to fix? No
 
Requested priority: High
 
Products/sites affected: ODB

**Comment by JasonGore** [comment]
I've confirmed this is an issue with Edge, but even the normal announcement of matches doesn't seem to work consistently, at least after the first announcement.

FWIW I've confirmed this behavior seems to work a lot better with Narrator + Chrome, and even announces when no results are found:
![pickers](https://user-images.githubusercontent.com/26070760/52756003-c78c7300-2fb4-11e9-8253-ae8c38f23a99.gif)

**Comment by HalcyonPride** [comment]
@JasonGore, thanks for triaging this! Would you be able to file a Bug on Edge? If not, I can show you how in an email. After which, please feel free to close this Bug as 'External' or something equivalent.

**Comment by JasonGore** [comment]
I've triaged it to the Picker team for now.. I wanted them to confirm the final triage. 😄

**Comment by JasonGore** [comment]
Also, please feel free to file bugs in Edge as well. Ultimately we're responsible for Fabric, not Edge. :)

**Comment by HalcyonPride** [comment]
@JasonGore, sounds good! And absolutely; I understand. I've filed Bugs on Edge before - I just incorrectly triaged this as a Fabric bug. 😬

**Comment by lukehumphrey** [comment]
Hi guys - this bug was flagged for us in an accessibility usability validation assessment and will currently block us from passing this validation (MS Project). 
If this is an Edge bug, can you let me know bug #/details ? lukehum@microsoft.com.
Thanks!

**Comment by HalcyonPride** [comment]
Hi @lukehumphrey, @JasonGore left this Bug for your team to triage - we haven't taken any action on it.

**Comment by JasonGore** [comment]
I'm still looking into this. There seems to be quite a simple `alert` usage here that isn't announced, so I suspect this is either an Edge or Narrator bug. I don't see any existing bugs in their repos that quite match, so I'm working on a minimal repro to report.

**Comment by JasonGore** [comment]
I have a [minimal repo CodePen](https://codepen.io/jasongore/pen/NJYQVV) that repros this issue in Edge but works fine in Chrome. I'll create an Edge issue for this. It's possible this is a repro of another bug I'm seeing in the Edge repo, but it is very technical so I am not sure. I'll let you know either way.

**Comment by JasonGore** [comment]
I've identified this bug in Edge's repo and they plan to fix it in UIA for Chromium. Since this is working in other browsers I will resolve this as external.

**Comment by HalcyonPride** [comment]
As described above - thanks, @JasonGore!

**Comment by HalcyonPride** [comment]
Hi @JasonGore, could you please help me remove the "Needs: Attention" tag? 😅
